src/scratchblocks.js

```diff
@@ -1699,34 +1698,37 @@ var scratchblocks = function () {
       c: 'color',
     }[part[1]];
 
-    var value = value ? ""+value : "";
+    var localValue = value;
```

Comment:
I don't think localValue is the best name here, however I have no better suggestions. Any reason we can't reuse value?